travis/test.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+if [ "$ARCH" == "" ]; then
+    echo 'Error: $ARCH is not set'
+    exit 1
+fi
+
+if [ "$ARCH" != "x86_64" ]; then
+    echo 'NOTICE: $ARCH is not supported yet'
+    exit 0
+fi
+
+TARGET=$1
+if [ "$TARGET" == "" ]; then
+    echo 'Error: $TARGET is not set'
+    exit 1
+fi
+
+
+# use RAM disk if possible
+if [ "$CI" == "" ] && [ -d /dev/shm ]; then
+    TEMP_BASE=/dev/shm
+else
+    TEMP_BASE=/tmp
+fi
+
+BUILD_DIR=$(mktemp -d -p "$TEMP_BASE" linuxdeploy-plugin-qt-build-XXXXXX)
+
+cleanup () {
+    if [ -d "$BUILD_DIR" ]; then
+        rm -rf "$BUILD_DIR"
+    fi
+}
+
+trap cleanup EXIT
+
+
+wget -N https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-"$ARCH".AppImage
+export LINUXDEPLOY_BIN=${PWD}/linuxdeploy-"$ARCH".AppImage
+chmod +x ${LINUXDEPLOY_BIN}
+
+pushd "$BUILD_DIR"
+
+git clone --depth=1 https://github.com/linuxdeploy/linuxdeploy-plugin-qt-examples.git
+
+## Install additional build stuff
+sudo apt-get install -y software-properties-common
+sudo add-apt-repository -y 'ppa:beineri/opt-qt-5.10.1-trusty'
+sudo apt update
+sudo apt install -y qt510-meta-minimal qt510declarative qt510webengine mesa-common-dev
+
+export QT_PREFIX=/opt/qt510
```

Comment:
Can we use `${QT_PREFIX:-/opt/qt510\`? That way, a developer can easily try newer/older/different versions of Qt without relying on Travis CI. Heck, we could even test multiple versions on Travis in parallel.